src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import { IonContent } from '@ionic/react'
+import Typography from '@material-ui/core/Typography'
+import { ThemeProvider } from '@material-ui/core/styles'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import theme from '../../styles/muiTheme'
+import Container from '@material-ui/core/Container'
+import Grid from '@material-ui/core/Grid'
+import Box from '@material-ui/core/Box'
+import Paper from '@material-ui/core/Paper'
+import ChatBubbleOutlineIcon from '@material-ui/icons/ChatBubbleOutline'
+import RestaurantIcon from '@material-ui/icons/Restaurant'
+import FingerprintIcon from '@material-ui/icons/Fingerprint'
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles'
+import Divider from '@material-ui/core/Divider'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Link from '@material-ui/core/Link'
+import CreateRoomForm from './CreateRoomForm'
+
+const AdminDashboard = () => {
+    const preventDefault = (event: React.SyntheticEvent) =>
+        event.preventDefault()
+
+    const useStyles = makeStyles((theme: Theme) =>
+        createStyles({
+            icon: {
+                marginLeft: theme.spacing(2),
+            },
+            textRight: {
+                textAlign: 'right',
+            },
+            list: {
+                paddingTop: 0,
+                paddingBottom: 0,
+            },
+            listItem: {
+                paddingTop: 0,
+                paddingBottom: 0,
+            },
+        })
+    )
+
+    const classes = useStyles()
+
+    return (
+        <ThemeProvider theme={theme}>
+            <CssBaseline />
+            <IonContent style={{ '--background': 'white' }}>
```

Comment:
to remove?

Reply:
For the moment the mobile background is something like `blue`. That page in particular needs to have a white background, although there's surely another more appropriate way to override that